src/routes/profile/profile.css

```diff
@@ -0,0 +1,35 @@
+.profile-wrapper {
+ max-width: 360px;
+ margin: 0 auto;
+ display: flex;
+ align-items: center;
+ background: #ffffff;
```

Comment:
Do you have prettier installed?

indent required for inner part, if you have prettier installed, alt-shift-f will fix this